code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm

```diff
@@ -0,0 +1,442 @@
+		mychild.health = mychild.maxHealth
+	else
+		SSblackbox.record_feedback("tally", "AI controlled Elite win", 1, mychild.name)
+	if(times_won == 1)
+		mychild.playsound_local(get_turf(mychild), 'sound/magic/cult_spell.ogg', 40, 0)
+		to_chat(mychild, "<span class='warning'><As the life in the activator's eyes fade, the forcefield around you dies out and you feel your power subside.\n\
+			Despite this inferno being your home, you feel as if you aren't welcome here anymore.\n\
+			Without any guidance, your purpose is now for you to decide.</span>")
+		to_chat(mychild, "<b>Your max health has been halved, but can now heal by standing on your tumor. Note, it's your only way to heal.\n\
+			Bear in mind, if anyone interacts with your tumor, you'll be resummoned here to carry out another fight. In such a case, you will regain your full max health.\n\
+			Also, be weary of your fellow inhabitants, they likely won't be happy to see you!</b>")
+		to_chat(mychild, "<span class='big bold'>Note that you are a lavaland monster, and thus not allied to the station. You should not cooperate or act friendly with any station crew unless under extreme circumstances!</span>")
+
+/obj/item/tumor_shard
+	name = "tumor shard"
+	desc = "A strange, sharp, crystal shard from an odd tumor on Lavaland. Stabbing the corpse of a lavaland elite with this will revive them, assuming their soul still lingers. Revived lavaland elites only have half their max health, but are completely loyal to their reviver."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "crevice_shard"
+	throwforce = 5
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 5
+
+/obj/item/tumor_shard/afterattack(atom/target, mob/user, proximity_flag)
+	. = ..()
+	if(istype(target, /mob/living/simple_animal/hostile/asteroid/elite) && proximity_flag)
+		var/mob/living/simple_animal/hostile/asteroid/elite/E = target
+		if(E.stat != DEAD || E.sentience_type != SENTIENCE_BOSS || !E.key)
+			user.visible_message("It appears [E] is unable to be revived right now. Perhaps try again later.")
+			return
+		E.faction = list("\ref[user]")
+		E.friends += user
+		E.revive()
+		user.visible_message("<span class='notice'>[user] stabs [E] with [src], reviving it.</span>")
+		E.playsound_local(get_turf(E), 'sound/magic/cult_spell.ogg', 40, 0)
+		to_chat(E, "<span class='userdanger'>You have been revived by [user]. While you can't speak to them, you owe [user] a great debt.  Assist [user.p_them()] in achieving [user.p_their()] goals, regardless of risk.</span>")
+		to_chat(E, "<span class='big bold'>Note that you now share the loyalties of [user].  You are expected not to intentionally sabotage their faction unless commanded to!</span>")
+		if(user.mind.special_role)
+			E.maxHealth = 300
+			E.health = 300
+		else
+			E.maxHealth = 200
+			E.health = 200
+			E.revive_cooldown = TRUE
+		E.desc = "[E.desc]  However, this one appears appears less wild in nature, and calmer around people."
```

Comment:
again, examine override. could use the factions variable?

Reply:
examine overide done